src/Tools/BuildValidator/LocalReferenceResolver.cs

```diff
@@ -44,12 +49,33 @@ public static DirectoryInfo GetNugetCacheDirectory()
             return new DirectoryInfo(nugetPackageDirectory);
         }
 
+        public static DirectoryInfo[] GetRefAssembliesDirectories()
+        {
+            // PROTOTYPE
```

Comment:
This isn't covered in #51097

Reply:
Oops, have added PROTOTYPE comments to the issue.